src/applications/vaos/utils/selectors.js

```diff
@@ -12,7 +12,18 @@ import {
   TYPES_OF_EYE_CARE,
   FETCH_STATUS,
 } from './constants';
-import { getRootOrganization } from '../services/organization';
+import {
+  getRootOrganization,
+  getSiteIdFromOrganization,
+  getIdOfRootOrganization,
+} from '../services/organization';
+import { getParentOfLocation } from '../services/location';
+
+// Only use this when we need to pass data that comes back from one of our
+// services files to one of the older api functions
+function parseFakeFHIRId(id) {
```

Comment:
I noticed a similar function in 5 different files.  Would it be worth creating a single util somewhere? I might have mentioned this before, so apologies if you already provided reasoning against it.

Reply:
Heh, my reasoning is that these are temporary and I want them to standout as bad/weird code so no one thinks they're supposed to be commonly used. All uses of them will go away when we're through all the mapping and update tickets. I'm removing a bunch of them in my in progress ticket, too.